### DISCUSSION TITLE
The order of all races labeled "Judging" is the same as that of all races labeled "Completed".

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [solodit/feedback#36] [Bug Report] [answered]
Hey hans,

In the "Audits" feature, matches labeled "Judging" should be above matches labeled "Completed" and below matches labeled "Live"!

Example:
![image](https://github.com/solodit/feedback/assets/33568293/69b326f2-d146-4d22-998c-6c455e3cdc64)
![image](https://github.com/solodit/feedback/assets/33568293/ac2e8030-9c00-4086-897d-adc58b5a6ca5)

**Comment by hans-cyfrin** [comment]
Sure! We will fix it very soon. Thank you for pointing out!

**Comment by wang1252** [comment]
@IAm0xAxe 

We fixed the problem.
Thank you for your feedback.

**Comment by IAm0xAxe** [comment]
@hans-cyfrin  @wang1252  

Thanks to your fixes, I'm already seeing changes. ^_^

Just now, I would like to make a suggestion again: I found two problems with the way contests are loaded.

Problem 1: When I check only the "Sherlock" platform, I find that only 20 Contests are loaded. This does not match the actual number(I guess it's the same problem with the other two platforms.).
![image](https://github.com/solodit/feedback/assets/33568293/c1c0567c-4f08-4068-9be9-6ef91b4e3c98)
![image](https://github.com/solodit/feedback/assets/33568293/e8cb4872-8457-4a39-8a8b-deadc55152f4)

Problem 2: The display of Contests, I would recommend using the page count method to display all the Contests, so that it is more convenient to search and query. For example: https://app.auditcrew.xyz/sherlock/contests
![image](https://github.com/solodit/feedback/assets/33568293/1c5c4198-09b8-4b07-9a0c-d4fe2fb0fc5e)

**Comment by wang1252** [comment]
@IAm0xAxe 

We are using infinite scroll for audit list view.
If you scroll down, it will loads continuously.
For the pagination, we will discuss about it.

**Reply by IAm0xAxe** [reply]
Using infinite scrolling can be very cumbersome when there are too many matches. The experience is not good. You can try to experience the paging feature. It's like the paging feature for audit reports, which makes it easier to retrieve.

**Reply by wang1252** [reply, marked as answer]
@IAm0xAxe 

We accepted your suggestion and updated to use pagination.
Thank you for your suggestion.

**Reply by IAm0xAxe** [reply]
Thank you for taking my advice. Thanks again for all your hard work.
